src/App.scss

```diff
@@ -1 +1,16 @@
 // style go here
+.title {
+  margin: 0;
+  text-align: center;
+  color: blue;
+}
```

Comment:
If you apply styles to a component, it should be in separate file for this component.
For this task it is not important because it is rather small, but for bigger projects try to follow this rule.